mysite/ShareBox/views.py

```diff
@@ -1,7 +1,38 @@
 from django.http import HttpResponse
+from ShareBox.models import Snippet
+from django.views.decorators.csrf import csrf_exempt
 import requests
 import httplib
+import json
 
 
-def send_data(request, snippet_data):
-    pass
+@csrf_exempt
+def insert_db(request):
+    if request.method == "POST":
+        body_unicode = request.body.decode('utf-8')
+        body = json.loads(body_unicode)
+        name = body['snippet_name']
+        privacy = body['snippet_privacy']
+        expiry = body['snippet_expiry']
+        snippet_data = body['snippet_file']
+        new_snippet_box, created = Snippet.objects.get_or_create(
+            snippet_name=name, snippet_privacy=privacy, snippet_expiry=expiry,
+            snippet_file=snippet_data)
+        new_snippet_box.save()
+        output = {}
+        url = "http://www/shareboxes.herokuapp.com/snippet/id/" \
+            + str(new_snippet_box.id)
+        output["url"] = url
+        print url
+    return HttpResponse(json.dumps(output))
```

Comment:
This is not completely correct. You need to set the `content-type` header as well. The right way to do this in Django can be found here: http://stackoverflow.com/questions/2428092/creating-a-json-response-using-django-and-python